src/implementations/vanilla/components/breadcrumb-harmonised/breadcrumb-harmonised.scss

```diff
@@ -9,10 +9,10 @@
 @use '@ecl/theme-dev/maps/typography';
 
 $_background-color: transparent;
-$_background-color-hover: map.get(color.$color, 'blue-5');
```

Comment:
I find the approach of having both ec/eu pairs in 1 theme sub-optimal. It's imposing an imperative mode which opens the door for possible blow in complexity and duplication

In the current setup in `next-v3` the imperative specificity is organized to be in 1 place in the (ec|eu)-core themes rather than the components.

Following the example we have with the globals, `src/themes/eu-core/_index.scss`:

```scss
@use '@ecl/theme-dev/globals' with (
  $border-radius: 'yes'
);
@use '@ecl/theme-dev/maps/color' with (
  $color: (
    'blue-5': #f3f6fc,
    'blue-10': #e7edfa,
    'blue-20': #cfdaf5,
    'blue-40': #9fb5ea,
    'blue-60': #6e91e0,
    'blue-80': #3e6cd5,
    'blue-100': #0e47cb,
    'blue-110': #0d40b7,
    'blue-120': #0b39a2,
    'blue-130': #0a328e,
    'blue-140': #082b7a,
    'yellow-10': #fffae6,
    'yellow-20': #fff5cc,
    'yellow-40': #ffeb99,
    'yellow-60': #ffe066,
    'yellow-80': #ffd633,
    'yellow-100': #fc0,
    'yellow-110': #e6b800,
    'yellow-120': #cca300,
    'yellow-130': #b38f00,
    'yellow-140': #b38f00,
    'grey-5': #f4f5f5,
    'grey-10': #e9eaeb,
    'grey-20': #d4d5d7,
    'grey-40': #a8aaaf,
    'grey-60': #7d8088,
    'grey-80': #515560,
    'grey-100': #262b38,
    'grey-110': #222732,
    'grey-120': #1e222d,
    'grey-130': #1b1e27,
    'grey-140': #171a22,
    'purple-100': #510dcd,
  )
);
```

I.e. the specificity comes by override on a theme level.

This approach would mean that `map.get(color.$color, 'blue-5')` here will be replaced with `#f3f6fc` (instead of ` rgb(242, 245, 249)`) at `src/presets/eu-core/src/eu-core.scss`

```scss
@use '@ecl/theme-eu-core';
@use '@ecl/preset-dev/src/dev';
```

The theme layer being first will replace tokens with their ec/eu specific values and then compile the component-specific code based on the tokens' specific values.

Reply:
Interesting.. So basically, the default map file would contain the common values, and the specific ones (EC/EU) would be defined in the theme file, right?
If tomorrow we have other themes, we may have to duplicate these values, but it shouldn't be a big overload.

Reply:
But the tokens are not the same between EC and EU (for instance there is no yellow-20 in EC, and no yellow-25 in EU.
So in the end we would have to have 2 separate scss files in components in any case I think

Reply:
Missing tokens between themes can go into the dev becoming defaults. It's up to design rules for the tokens to not be used in the wrong places in the components

Reply:
The thing is, we don't know what would be the differences between EC and EU. That may be more than just a colour swap. For instance, there will probably be cases where there is a background colour on EU, and not on EC.
So I don't know how we could handle this kind of thing without having 2 scss files in the component

Reply:
It's true we lack long-term vision. For now we know there is a difference in colors and we can handle them.

There's a card we can play before splitting files: having system-aware context variable passed to scss. Here's the ["can do" commit](https://github.com/ec-europa/europa-component-library/pull/1692/commits/c91aa7eab1213f0054332fd8462d9c959cc394b9#diff-37476fe9fa6e6d92477a0c7c7f0672ca07dab5685faee067d4c4c47889f29c0d).

If the difference is in a property or two in a given file, we can still use this approach and set specifics conditionally without having multiple files.